### DISCUSSION TITLE
Inicialização de constantes no C#

### DISCUSSION THREAD

**Discussion by WinterDP** [balta-io/forum#671] [Tira Dúvidas] [answered]
## Aula: Linguagem de Programação com C# :  Constantes
Olá,
Estou realizando o curso Fundamentos do C# e esbarrei em uma dúvida.
Durante a aula foi colocado que é possível inicializar variáveis constantes sem o valor explícito , pois elas seriam inicializadas com um valor padrão
![image](https://github.com/user-attachments/assets/3ba39fd4-3ba1-4f53-9171-020b5b014e03)
![image](https://github.com/user-attachments/assets/2b73c3c2-008f-4019-9dbe-6cd7143f1c21)

Contudo, ao realizar o processo para teste recebi o seguinte erro de compilação:
`error CS0 145: O campo const requer um valor a ser fornecido`
E ao atribuir valor a constante o erro é corrigido.

já foi possível a atribuição implícita e o erro é presente nas novas versões do C#/.NET? 
Vi uma discussão semelhante levantada em #529, mas a abordagem da correção divergiu.

**Versão .Net: 8.0.303**

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado
https://github.com/balta-io/forum/discussions/529
